js/src/forum/utils/getCleanDisplayName.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Whether to use the old mentions format.
+ *
+ * `'@username'` or `'@"Display name"'`
+ */
+export const ShouldUseOldFormat = () => app.forum.data.attributes.allowUsernameMentionFormat || false;
+
+const GetDeletedUserText = () => app.translator.trans('core.lib.username.deleted_text');
+
+/**
+ * Fetches a user's username or display name.
+ *
+ * Chooses based on the format option set in the admin settings page.
+ *
+ * @param user An instance of the User model to fetch the username for
+ * @param useDisplayName If `true`, uses `user.displayName()`, otherwise, uses `user.username()`
+ */
+export default function cleanDisplayName(user, useDisplayName = true) {
```

Comment:
Why the rename?

Reply:
Prior to this, the function accepted a username input to clean up, but now the function fetches the username.

I didn't feel that cleanDisplayName was a good explanation, and that getCleanDisplayName was more clear.

Reply:
Let's change the file name then?